custom-fields/custom-field-write.component.html

```diff
@@ -1 +1,11 @@
-<input class="custom-title-input" [(ngModel)]="value" />
+<sf-input
+    [name]="settings.key"
+    [look]="settings.look"
+    [placeholder]="settings.placeholder"
+    [recommendedCharacters]="settings.recommendedCharacters"
+    #inputField
+    ngDefaultControl
+    [(ngModel)]="value"
+    (onBlur)="onFocusOut()"
```

Comment:
onBlur instead of onFocusOut! otherwise error is thrown because onFocusOut is not defined in the component

Reply:
fixed it :)